provider/maas/config_test.go

```diff
@@ -15,7 +14,7 @@ import (
 
 // Ensure MAAS provider supports the expected interfaces.
 var (
-	_ config.ConfigSchemaSource = (*maasEnvironProvider)(nil)
+	_ config.ConfigSchemaSource = (*MaasEnvironProvider)(nil)
```

Comment:
As long as the implementation is used as an instance of the interface in the same repo, I think that's fine.   That being said, they're also not really costing us anything.